.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+* @abhinavrastogi-harness @knagurski
+
+/packages/* @abhinavrastogi-harness @knagurski
+
+/apps/gitness/* @abhinavrastogi-harness @vardanbansal-harness
```

Comment:
let's add kevin as the owner for portal app as well

Reply:
Done